src/global/helpers/overlay-trigger-attributes.ts

```diff
@@ -29,3 +29,41 @@ export function removeAriaOverlayTriggerAttributes(trigger: HTMLElement): void {
   trigger.removeAttribute('aria-controls');
   trigger.removeAttribute('aria-expanded');
 }
+
+/**
+ * Add meaningful aria attributes to trigger element of role='combobox'
+ */
+export function setAriaComboBoxAttributes(
+  trigger: HTMLElement,
+  overlayId: string,
+  expanded: boolean
+): void {
+  if (!trigger) {
+    return;
+  }
+
+  trigger.setAttribute('autocomplete', 'off');
```

Comment:
Chromium has decided that `autocomplete="off"` can be ignored and a native autocomplete will still be shown. The only way to reliably suppress it is via `new-password`.

Reply:
Apparently the 'new-password' prevent autofill but not suggestions from the browser.
In our case, I think that 'off' works "better"